libpod/healthcheck_linux.go

```diff
@@ -46,6 +47,17 @@ func (c *Container) createTimer() error {
 	return nil
 }
 
+// Wait for a message on the channel.  Throw an error if the message is not "done".
+func systemdOpSuccessful(c chan string) error {
+	msg := <-c
+	switch msg {
+	case "done":
```

Comment:
is it guaranteed to be lower case "done"?

Reply:
Yes, that's guaranteed.